libs/platform/src/lib/components/form/multi-input/multi-input.component.ts

```diff
@@ -96,22 +100,21 @@ export class PlatformMultiInputComponent extends BaseMultiInput implements OnIni
         this._disabled = value;
     }
 
-    /** Represents the  value injected from the selected list value*/
     @Input()
-    set selectedListVariable(selectedValue) {
-        this._selectedListVariable = selectedValue;
-        if (this._selectedListVariable) {
-            this.addToArray(this._selectedListVariable);
-        }
+    get status(): Status {
+        return this._state;
     }
-    get selectedListVariable(): any {
-        return this._selectedListVariable;
+    set status(value: Status) {
```

Comment:
we have not used FormFieldControl status. we are setting status of our own. This is not using platform form behavior.